hack/cherry_pick_pull.sh

```diff
@@ -84,8 +84,10 @@ shift 1
 declare -r PULLS=( "$@" )
 
 function join { local IFS="$1"; shift; echo "$*"; }
-declare -r PULLDASH=$(join - "${PULLS[@]/#/#}") # Generates something like "#12345-#56789"
-declare -r PULLSUBJ=$(join " " "${PULLS[@]/#/#}") # Generates something like "#12345 #56789"
+PULLDASH=$(join - "${PULLS[@]/#/#}") # Generates something like "#12345-#56789"
+declare -r PULLDASH
```

Comment:
change reason:
```
In hack/cherry_pick_pull.sh line 87:
declare -r PULLDASH=$(join - "${PULLS[@]/#/#}") # Generates something like "#12345-#56789"
           ^-- SC2155: Declare and assign separately to avoid masking return values.
```